src/audio/drc/drc_math_generic.c

```diff
@@ -118,7 +118,7 @@ inline int32_t drc_sin_fixed(int32_t x)
 }
 
```

Comment:
Your commit is doing two things when it should do only one! 

Please move comments updates on a separate patch.